auth-manager/src/main/java/com/qpay/authmanager/model/dto/UserModification.java

```diff
@@ -0,0 +1,19 @@
+package com.qpay.authmanager.model.dto;
+
+import com.qpay.libs.models.UserType;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Builder;
+
+@Builder
+public record UserModification (
+
+        @NotBlank(message = "Email is required")
+        String email,
+
+        @NotBlank(message = "Password is required")
+        String password,
+
+        @NotBlank(message = "UserType is required")
```

Comment:
Is it expected to update user type?
If not, it can not be required. Or split DTOs